docs/documentation/layout/hero.html

```diff
@@ -469,12 +469,12 @@ <h2 class="subtitle">
   </div>
 
 {% highlight html %}
-<section class="hero is-link is-fullheight">
+<section class="hero is-link is-fullheight-with-navbar">
```

Comment:
This class is only used when the navbar is fixed.

Reply:
Sorry-- perhaps I'm not understanding something.

I believe this line is the code-snippet corresponding to the example right above it. Then shouldn't this line match with line 460 above?

Reply:
Ah yes of course! I missed that. Sorry, you're right.